src/Controller/Api/DatasetSubmissionController.php

```diff
@@ -2,23 +2,27 @@
 
 namespace App\Controller\Api;
 
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
-use Symfony\Component\Form\FormInterface;
+use App\Entity\File;
+use App\Entity\DatasetSubmission;
+use App\Entity\Fileset;
 
-use Symfony\Component\Routing\Annotation\Route;
+use App\Util\UrlValidation;
+
+use App\Form\DatasetSubmissionType;
+
+use Doctrine\ORM\EntityManagerInterface;
 
 use FOS\RestBundle\Controller\Annotations\View;
 
 use Nelmio\ApiDocBundle\Annotation\Operation;
 use Nelmio\ApiDocBundle\Annotation\Model;
+
 use Swagger\Annotations as SWG;
 
-use App\Util\UrlValidation;
-use App\Entity\DatasetSubmission;
-use App\Entity\Fileset;
-use App\Form\DatasetSubmissionType;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
Cool ordering, but in the past we have always put our own App last, and we should be consistent.

Reply:
I thought it was alphabetical, will move App stuff to the bottom